### PR TITLE
#502 Remove params for setTimeout in behaviour

### PR DESCRIPTION
**Pull request by jevenson**
#502 All tests passing, works in IE 9 and other browsers.

I see this code was modified in PR #530 so adjustments will need to be made.

### CODE REVIEW

**Comment by jevenson** [comment]
Here is some more information I found on the issue.
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout#Polyfill

File: src/jquery.mask.js [hunk 1]
```diff
@@ -217,7 +217,7 @@
                     var newVal   = p.getMasked(),
                         caretPos = p.getCaret();
 
-                    setTimeout(function(caretPos, newVal) {
+                    setTimeout(function() {
                       p.setCaret(p.calculateCaretPosition(caretPos, newVal));
                     }, 10, caretPos, newVal);
```
Comment (jevenson) [inline src/jquery.mask.js]: Actually, it looks like these two params at the end can be removed as well.

**Comment by jevenson** [comment]
@igorescobar Any ETA on when this could be reviewed?

**Comment by igorescobar** [comment]
Fixed by PR #556. Thanks!

**Pull request closed without merging**